magprime/templates/regextra.html

```diff
@@ -77,9 +77,21 @@
         {% endif %}
 
         // MAGFest uses tiered badge pricing based on caps, so we want to link to our FAQ explaining this
-        if ($('.prereg-price-notice').size()) {
-            $('#reg-types').append("<div class='help-block col-sm-9 col-sm-offset-3'>Prices may increase before the set date. {{ macros.popup_link("http://magfest.org/faq#-KOpcFnNhdsUIqvvf6VB", "Why is this?")|e }}</div>");
-        }
+        {% if c.BADGES_LEFT_AT_CURRENT_PRICE %}
+            {% if c.BADGES_LEFT_AT_CURRENT_PRICE == -1 %}
+                {% set badges_left_text = 'Limitless' %}
+            {% elif c.BADGES_LEFT_AT_CURRENT_PRICE <= 100 %}
+                {% set badges_left_text = 'Critical' %}
```

Comment:
I actually like "Almost Gone" better then "Critical". I know it's not exactly what Nick asked for, but I think it your original phrasing was better here.